.github/workflows/check-examples.yml

```
@@ -0,0 +1,60 @@
+name: Check example notebooks
+
+on:
+  pull_request:
+    branches:
+      - main
+      - develop
+  schedule:
+    # 04:00 every Saturday morning
+    - cron: '0 4 * * 6'
+
+env:
+  PYTKET_REMOTE_QUANTINUUM_USERNAME: ${{ secrets.PYTKET_REMOTE_QUANTINUUM_USERNAME }}
+  PYTKET_REMOTE_QUANTINUUM_PASSWORD: ${{ secrets.PYTKET_REMOTE_QUANTINUUM_PASSWORD }}
+
+jobs:
+
+  changes:
+    runs-on: ubuntu-20.04
+    outputs:
+      examples: ${{ steps.filter.outputs.examples }}
+    steps:
+    - uses: actions/checkout@v3
+    - uses: dorny/paths-filter@v2
```

```suggestion
    - uses: dorny/paths-filter@v2.10.2
```
Generally better to specify exact version for external actions when possible.